power.c

```diff
@@ -0,0 +1,21 @@
+//Emma L
```

Comment:
Your code compiled, but didn't properly follow directions - be careful next time.